de-de/c-de.html.markdown

```diff
@@ -0,0 +1,880 @@
+---
+language: c
+filename: learnc-de.c
+contributors:
+    - ["caminsha", "https://github.com/caminsha"]
+lang: de-de
+---
+
+Ach, C. Immer noch **die** Sprache für modernes High-Performance Computing.
+
+C ist wahrscheinlich die niedrigste Programmiersprache, welche die meisten
+Programmierer je brauchen werden. Die Geschwindigkeit von C ist enorm, allerdings
+muss man sich stets der maneullen Speicherverwaltung bewusst sein.
+
+
+> **Über Compiler Optionen**
+> 
+> Standardmässig sind `gcc` und `clang` ziemlich ruhig bezüglich Warnungen und 
+> Fehlern, obwohl dies sehr nützliche Informationen sein können. Es wird
+> empfohlen, strengere Compiler Optionen zu verwenden. Hier sind einige empfohlene
+> Standards:
+> `-Wall -Wextra -Werror -O2 -std=c99 -pedantic`
```

Comment:
should we stick to c99 or take a more recent standard?

Reply:
Good question. I've heard that code written in C99 is more portable.

What do you think? Would C11 be better?

Reply:
I cannot make that decision, probably project dependent. 

I believe it would be sensible to explain to provide a link to a resource that explains what those flags do. As things currently stand it is a bit confusing. Maybe this one? https://stackoverflow.com/questions/3375697/useful-gcc-flags-for-c